tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Handles logging functionality.
+"""
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority, publisher_id, msg):
```

Comment:
Rather than passing 'msg' you should pass the payload as it is not the same everywhere. It is different for jobs, logs etc